notification-server/src/models.rs

```diff
@@ -0,0 +1,48 @@
+use concordium_rust_sdk::base::contracts_common::AccountAddress;
+use enum_iterator::Sequence;
+use num_bigint::BigInt;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct NotificationInformation {
```

Comment:
Would be nice to have some comments on the fields and the overall types in follow-up PRs (when this type is finalized).